queries/smali/highlights.scm

```diff
@@ -82,7 +82,13 @@
   (#lua-match? @keyword.return "^return"))
 
 ((opcode) @conditional
-  (#match? @conditional "^(if|cmp)"))
+  (#lua-match? @conditional "^if"))
+
+((opcode) @conditional
+  (#lua-match? @conditional "^cmp"))
+
+((opcode) @exception
+  (#lua-match? @exception "^throw"))
```

Comment:
Duplicate?
```suggestion
```

Reply:
aha, I copied & pasted w/o replacing it w/ if